### DISCUSSION TITLE
Are batches supported?

### DISCUSSION THREAD

**Discussion by alinsavix** [aatikturk/obsws-python#65] [Q&A] [answered]
Hi,

Does obsws-python support batch requests? There's only one place in the source that batches are mentioned (in the documentation for sleep()), which implies both that batches aren't supported (because they aren't otherwise mentioned), and that they are supported (because sleep() isn't useful outside of batches).

Could you please clarify? Thanks!

**Comment by aatikturk** [comment, marked as answer]
Hi,

Batch requests are not supperted in `obsws-python`.  We used [obswebsocket documentation](https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#requests) to implement `ReqClient` methods. The reason we included the `sleep` method in the library is, if users want to implement their own solutions to process batch requests and need a delay function, it may help.

Thanks.
